src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.Stream.cs

```diff
@@ -13,16 +13,39 @@ namespace System.Net.Quic;
 public partial class QuicStream : Stream
 {
     // Seek and length.
+    /// <inheritdoc />
+    /// <summary>Gets a value indicating whether the <see cref="T:System.Net.Quic.QuicStream" /> supports seeking.</summary>
```

Comment:
Prefix and full name not needed in triple slash. The porting tool will take care of adding them for dotnet-api-docs (they are required there).
```suggestion
    /// <summary>Gets a value indicating whether the <see cref="QuicStream" /> supports seeking.</summary>
```